new-frontiers-14/frontier-station-14
# LVHI Broom Shuttle (New)

## About the PR
### New shuttle:
- Micro shuttle for round start janitors. The shuttle can be bought from Outpost Staff Shipyard Console at Frontier Outpost by Janitor/SR. Cheap, but has all the tools a janitor might need to perform his duties outside Frontier Outpost.

### Bundled changes:
- Renamed SR Shipyard Console -> Outpost Staff Shipyard Console and moved it to the corner by the Dock 6c.
- Moved mail carrier shuttles to Outpost Staff Shipyard Console, removed Mail Carrier Shipyard Console.
- Mapped additional Station Beacons to the Outpost.
- Mapped cell recharger and recharger to SR's Office. Removed wall-mounted recharger.
- Mapped materials wall locker to mail carrier room on Frontier.
- Moved grid .yml for mailpod and parcel to SR folder.
- Made fixtures on small gyro and small thruster smaller.

## Why / Balance
Cheap micro shuttle to encourage round start janitors to clean not only the Frontier Outpost, but other POI too.

## How to test
1. Spawn as janitor, insert your ID into Shipyard Console and buy LVHI Broom.
2. Spawn as contractor/pilot/mercenary, insert your ID into Shipyard Console, observe lack of LVHI Broom on the list.
3. Spawn any ID that has Janitor access level or above (SR/ERT/Sheriff), insert spawned ID into Shipyard Console and buy LVHI Broom.

## Media
### Shuttle
![2024-8-13_10 09 22](https://github.com/user-attachments/assets/583aa938-fe91-4e76-8f02-6814a2bfb89a)
### SR office changes
![2024-8-13_10 10 04](https://github.com/user-attachments/assets/d086d963-858d-4c23-9009-05ac669d7315)
### Mail Carrier room changes
![2024-8-13_09 42 43](https://github.com/user-attachments/assets/65351f2a-ff28-4a97-b153-19a1948cb155)

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- add: Added new micro shuttle LVHI Broom, available only to personnel with janitorial access from Outpost Staff Shipyard Console at Frontier Outpost.
- tweak: Placed Outpost Staff Shipyard Console in the nook by the Dock 6c (to the North from SR's office), , removed SR and Mail Carrier Shipyard Consoles from Frontier Outpost.
- tweak: Moved mail carrier shuttles and SR shuttle to Outpost Staff Shipyard Console.
